holmes/plugins/toolsets/datadog/toolset_datadog_metrics.py

```
@@ -149,7 +179,43 @@
+                                end=params.get("to_time"),
+                                default_time_span_seconds=DEFAULT_TIME_SPAN_SECONDS,
+                            )
+                            datadog_url = generate_datadog_metrics_url(
+                                self.toolset.dd_config,
+                                params.get("query", ""),
+                                from_time,
+                                to_time,
+                            )
+                            error_msg += f"\nView in Datadog: {datadog_url}"
+                        except Exception:
+                            pass  # Skip URL generation if timestamps can't be processed
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)